File: LeetCode/Arrays/JS/1470. Shuffle the Array.js
/**
 * @param {number[]} nums
 * @param {number} n
 * @return {number[]}
 */
var shuffle = function(nums, n) {
    let ans = [];
    for(let i=0;i<n;i++)
    {
        ans[2*i] = nums[i];
        ans[2*i+1] = nums[i+n];
    }
    return ans;
};

File: LeetCode/Arrays/C++/1470. Shuffle the Array.cpp
#include <bits/stdc++.h>
using namespace std;

class Solution {
public:
    vector<int> shuffle(vector<int>& nums, int n) {
        vector<int> ans;
        for(int i{};i<n;i++)
        {
            ans.push_back(nums[i]);
            ans.push_back(nums[i+n]);
        }
        return ans;
    }
};